pandas/core/arrays/boolean.py

```diff
@@ -294,6 +294,7 @@ class BooleanArray(BaseMaskedArray):
     # Fill values used for any/all
     _truthy_value = True
     _falsey_value = False
+    _NONE_VALUES = {"nan", "NaN", "None", "NA", "null", "NULL"}
```

Comment:
I fail to understand how this is related to having blanks. In the current version of pandas (without the changes in this PR), what's the value of the blank value in the spreadsheet when constructing the extension array? Feels like it should be `None` or an empty string, not one of those.

Reply:
@datapythonista Hello! Thanks for the review! The point here is that I added this list because the isna() fails. The None values are converted into strings so without this list, the "None" values will raise a ValueError. Of course if you have any suggestion on how to implement this better please tell me.